op-chain-ops/deployer/opsm/opchain.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
 )
 
+// PermissionedGameStartingAnchorRoots is a root of bytes32(hex"dead") for the permissioned game at block 0,
+// and no root for the permissionless game. It was generated with a console.log in DeployOpChain.s.sol
```

Comment:
I don't think we need to keep this detail around. 

```suggestion
// and no root for the permissionless game.
```